solana-programs/anchor/audius-data/lib/utils.ts

```diff
@@ -6,6 +6,9 @@ import * as secp256k1 from "secp256k1";
 import keccak256 from "keccak256";
 import { AudiusData } from "../target/types/audius_data";
 const { PublicKey } = anchor.web3;
+import { getEthContractAccounts } from "../../../../libs/initScripts/helpers/utils.js"
+import initAudiusLibs from "../../../../libs/examples/initAudiusLibs.js"
```

Comment:
I think this will break the build for packaging up this library as an npm package

Reply:
😅 ah yeah, converting to import from local env